dashboard/client/src/components/LogView/LogVirtualView.tsx

```diff
@@ -118,8 +189,25 @@ const LogVirtualView: React.FC<LogVirtualViewProps> = ({
   if (listRef) {
     listRef.current = outter.current;
   }
+  const [selectedLogLine, setSelectedLogLine] = useState<string>();
+  const handleLogLineClick = useCallback((logLine: string) => {
+    setSelectedLogLine(logLine);
+  }, []);
+
   const itemRenderer = ({ index, style }: { index: number; style: any }) => {
     const { i, origin } = logs[revert ? logs.length - 1 - index : index];
+
+    let message = origin;
+    let formattedLogLine = origin;
+    try {
+      const parsed_origin = JSON.parse(origin);
+      // Iff the parsed origin has a message field, use it as the message.
+      message = parsed_origin.message || origin;
```

Comment:
Can we also render the timestamp with every line if we have structured input?

Reply:
Yep, let me add an interpolation for it